src/CloseCircle.js

```diff
@@ -4,33 +4,35 @@ import React, {PropTypes} from 'react';
 const RECT_WIDTH = 1.5;
 const MARGIN = 8;
 
-const CloseCircle = (props) => {
-  const {
-    diameter,
-  } = props;
+class CloseCircle extends React.PureComponent {
```

Comment:
Functional components are actually a better choice. This was the result of a hasty solution to make the library work on Node 6.